src/components/Form/Checkbox.tsx

```diff
@@ -65,6 +65,8 @@ export interface CheckboxProps {
    * @default uuidV4
    */
   id?: string;
+  name?: string;
+  value?: string;
```

Comment:
Do we want to describe these as well?

Reply:
Yeah, that is a good question. As those are "basic/default" attributes for form element I was not sure if it is needed to have explicit documentation. 

As also other component, maybe including checkbox might extends some HtmlxxxElement and supports all the attributes from there if it is given to as prop, if I read it correctly.

Reply:
Yeah, I think it has the basic HMTL input props. I think either case (to document or not to document) could be argued. They are clearly something that a programmer using the component expects to be there. I guess the question is that do we want to signal specifically that we support these props.

Reply:
True. Maybe it could be good that we explicitly tell these `name` and `value` props in our documentation by providing the JSDoc for them. 

Should 
```typescript
/** Name **/
  name?: string;
/** Value **/
  value?: string
```
be enough?

Reply:
For that purpose I think those are just fine :)

Reply:
This is now changed.